### PR TITLE
Fixes #263: Created movie tweets review app

### LINKED ISSUE #263: Create Movie Tweets App
- Issue type: Feature request

### Short description
Add a new app which shows movie tweets with statistics.
In the app movie tweets will be shown according to emotion of the tweets i.e., positive or negative comments the movie gets. Its shows review/reaction of people if the movie is good or bad by just typing the name of the movie. Usually we follow few movie websites for review instead we can directly get review of people in this loklak movie review app.   

Am going to do this by  using search.json and also will try to add country wise emotion statistics using new classifier.json API.

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #263 
demo link: https://kavithaenair.github.io/movieReview/

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @singhpratyush @vibhcool @Achint08 Please review.

File: movieReview/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        <script src="js/script.js"></script>
+        <script src="js/highcharts.min.js"></script>
+        <script src="https://code.highcharts.com/highcharts-3d.js"></script>
+        <script src="js/exporting.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular.min.js"></script>
+
+
+        <style>
+            .post-wrapper:first-child .post-inner {
+            border-top-color: rgba(0, 0, 0, 0.1);
```
Comment (djmgit) [inline movieReview/index.html]: indentation is not proper.

File: movieReview/index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            }
+            .post-wrapper .post-inner {
+            background-color: white;
```
Comment (djmgit) [inline movieReview/index.html]: here too

File: movieReview/index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            color: #666666;
+            -webkit-transition-duration: 0.25s;
+            transition-duration: 0.25s;
+            -webkit-transition-timing-function: ease-out;
+            transition-timing-function: ease-out;
+            border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+            padding: 20px;
+            }
+            .post-wrapper .post-inner.open {
+            margin-top: 20px;
```
Comment (djmgit) [inline movieReview/index.html]: here too

File: movieReview/index.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            margin-bottom: 20px;
+            border-radius: 5px;
+            box-shadow: 0 4px 1px rgba(0, 0, 0, 0.2);
+            border-bottom-color: transparent;
+            }
+            .post-wrapper .post-inner.open:hover {
+            background-color: white;
```
Comment (djmgit) [inline movieReview/index.html]: here too

File: movieReview/index.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            .post-wrapper .post-inner p.lead {
+            font-size: 1.2em;
+            }
+            .Emoji.Emoji--forText {
+            width: 2%;
+            height: 2%;
+            padding: initial;
+            margin-left: 7px;
+            }
+        </style>
```
Comment (djmgit) [inline movieReview/index.html]: It will be nice if you move this css part to a separate file.

File: movieReview/index.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            <br></br>
+            <h1>Review for movie: {{query}}</h1>
+        <center><div ng-if="spinner && !tweetsStatus"><img src="loading.gif" height="400" width="400"/></div></center>
+        <div style="width: 100%;margin: 0 auto;">
+            <center>
+                <div style="width: 50%;height: 400px;float: center;" class="pie-chart"></div>
+            </center>
+        </div>
+            <div ng-if="tweetsStatus">
+                <div class="row" ng-repeat="value in tweetsStatus | limitTo:20">
```
Comment (djmgit) [inline movieReview/index.html]: any specific reason for setting a limit of 20 ? (just asking)

Comment (kavithaenair) [inline movieReview/index.html]: @djmgit , its just not to over-populate the screen.

**Comment by kavithaenair** [comment]
@djmgit Please re-review

**Comment by kavithaenair** [comment]
@djmgit yes sure. Thank you for pointing that out.

**Comment by kavithaenair** [comment]
@djmgit It's not logic error, checked it.

File: movieReview/index.html [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        <link href="css/bootstrap.min.css" rel="stylesheet">
+        <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+        <link rel="stylesheet" href="css/ie10-viewport-bug-workaround.css">
+        <!-- Custom styles for this template -->
+        <link href="css/starter-template.css" rel="stylesheet">
+        <script src="../../js/jquery.min.js"></script>
+        <script src="../../js/angular.min.js"></script>
+        <script src="js/script.js"></script>
+        <script src="js/highcharts.min.js"></script>
+        <script src="https://code.highcharts.com/highcharts-3d.js"></script>
```
Comment (mariobehling) [inline movieReview/index.html]: Avoid dependencies on additional external services.

File: movieReview/index.html [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+        <script src="js/exporting.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.1/angular.min.js"></script>
```
Comment (mariobehling) [inline movieReview/index.html]: Avoid dependencies on external services.

**Comment by kavithaenair** [comment]
@mariobehling Made the changes Please review.

**Comment by mariobehling** [comment]
What is the expected outcome now? What is the question you answer with this app? How is this app supposed to help people? What are the next steps in development?

**Comment by kavithaenair** [comment]
@mariobehling I think this app will help people to decide what the movie will be like and if they really want to watch it (the statistics is based on emotion quotient of the tweets), for example people can decide if the movie is of their kind (happy or sad etc). In the next step, the movie tweets can be aggregated according to the countries its mostly watched in which makes the decision more easier for the people using the app, as it will tell them the taste of people around them for that particular movie they are searching for.

**Comment by mariobehling** [comment]
>  I think this app will help people to decide what the movie

The movie? Testing this out, I don't get relevant results. Simply matching a search term with the word "movie"? Is this your app?

Who should be using this? -> There are no relevant results.

I thought in GSoC our goal is to develop something that gets actually used by people. Where is the plan to develop this app? It seems you just come up with some idea and then get started. We have serious challenges to solve and now we are spending time on a movie app where it is unclear if there will be any users at all.

**Comment by mariobehling** [comment]
> In the next step, the movie tweets can be aggregated according to the countries its mostly watched in which makes the decision more easier for the people using the app, as it will tell them the taste of people around them for that particular movie they are searching for.

Where do you get the data for that? We don't have the data. How do you manage languages? You want to do this in a few days and get this to be used by lots of people or is it just a small exercise to increase project contributions? The real problem - this is not thought-through.

**Comment by kavithaenair** [comment]
@mariobehling This app will help people who love to watch movies (something similar to IMDb but we here are using tweets emotion to judge the movie). And according to me, the aggregation I explained earlier will not be based on language basis but where the tweets are being posted from i.e., the location. 
Am sorry if this is not up to the mark, I will ask my co-developers to give in any suggestions to improvise this app and make sure it is useful for few set of people atleast.

**Comment by kavithaenair** [comment]
@mariobehling Can I proceed further with this app?

**Comment by mariobehling** [comment]
After the discussion, I think it is clear that the app will not have a focused output - meaning, it is not possible with your approach to match tweets exactly with movies and the quality of results is low with this approach. An option would be to match results with movie listings. However, this does not seem to be something where we should invest a lot of time now as there are many open issues with the server and possibilities to make use of twitter data in other ways. Therefore closing it. Please focus on priority issues.

**Pull request closed without merging**